dll/win32/msafd/misc/dllmain.c

```diff
@@ -2549,6 +2549,14 @@ WSPIoctl(IN  SOCKET Handle,
             ((SOCKET_ADDRESS_LIST*)lpvOutBuffer)->Address[0].iSockaddrLength = sizeof(Socket->SharedData->WSLocalAddress);
             ((SOCKET_ADDRESS_LIST*)lpvOutBuffer)->Address[0].lpSockaddr = &Socket->SharedData->WSLocalAddress;
 
+            Errno = NO_ERROR;
+            Ret = NO_ERROR;
+            break;
+        case SIO_UDP_CONNRESET:
+            /* FIXME: It's a fix not to fail with unimplemented
+               This control code shoute controls whether UDP 
```

Comment:
shoute? I don't know that word. Looks like a typo.

Reply:
Surely, it was a typo. Applied the suggestion in another commit. Thank you.